releases/v2.0-beta.20180305.md

```diff
@@ -0,0 +1,406 @@
+---
+title: What&#39;s New in v2.0-beta.20180305
+toc: false
+summary: Additions and changes in CockroachDB version v2.0-beta.20180305
+---
+
+## March 05, 2018
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180305.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180305.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180305.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180305.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Build Changes
+
+- CockroachDB now builds with go 1.9.4 and higher. [#22608][#22608]
+
+### General Changes
+
+- Diagnostics reports now include information about changed settings and statistics on types of errors encountered during SQL execution. [#22705][#22705], [#22693][#22693], [#22948][#22948]
+
+### Enterprise Edition Changes
+
+- Bulk IO rate limiting is now done per-store rather than per-node. [#22729][#22729]
+- Revision history backup and restore no longer considered experimental. [#22679][#22679]
```

Comment:
`is no longer`

Also, does `revision history` = `point-in-time`? If so, is `revision history` the term we're going with?